certif/app/components/enrolled-candidates.js

```diff
@@ -163,6 +165,11 @@ export default class EnrolledCandidates extends Component {
     this._handleSavingError(errorText, certificationCandidate);
   }
 
+  _handleEntityValidationError(certificationCandidate, err) {
+    const errorText = get(err, 'errors[0].detail');
+    this._handleSavingError(errorText, certificationCandidate);
```

Comment:
On pourrait supprimer la dependance à lodash, mais vu que c'est fait nul part ailleurs, autant laisser ainsi :D
```suggestion
    const errorText = err.errors?.[0].detail;
    this._handleSavingError(errorText, certificationCandidate);
```